.github/workflows/dep-update-review.yml

```diff
@@ -0,0 +1,72 @@
+name: Dependency Update Reviewer
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+    paths:
+      - "**Cargo.lock"
+      - "**Cargo.toml"
+
+jobs:
+  generate-report:
+    runs-on: ubuntu-latest-xl
+    name: Generate update review report
+    outputs:
+      update-report: ${{ steps.set-update-report.outputs.report }}
+      update-report-length: ${{ steps.set-update-report.outputs.length }}
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2.3.4
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: checkout base ref
+        uses: actions/checkout@v2.3.4
+        with:
+          ref: ${{ github.event.pull_request.base.sha }}
+          path: diem-base
+      - name: install depdive
+        run: cargo install --git https://github.com/diem/whackadep --branch main depdive
+      - name: Get depdive update report
+        id: set-update-report
+        run: |
+          output="$(depdive update-review paths ./diem-base/ ./)"
+          echo "${output}"
+          output="${output//'%'/'%25'}"
+          output="${output//$'\n'/'%0A'}"
+          output="${output//$'\r'/'%0D'}"
+          echo "::set-output name=report::${output}"
+          length=$(echo ${output} | wc -c)
+          echo "::set-output name=length::${length}"
+
+
+  make-pr-comment:
```

Comment:
There's no real benefit to having this broken into two separate jobs, lets merge them into a single job

Reply:
is there any harm/cost?

Reply:
you wouldn't need to go through the song and dance of passing values via `echo "::set-output"`